kart/tabular/table_dataset.py

```diff
@@ -165,44 +154,20 @@ def features(self, spatial_filter=SpatialFilter.MATCH_ALL, log_progress=False):
                 n_matched += 1
                 yield feature
 
-            if log_progress and n_chunk == self.NUM_FEATURES_PER_PROGRESS_LOG:
-                t = time.monotonic()
-                self._log_feature_progress(
-                    plog, n_read, n_chunk, n_matched, n_total, t0, t0_chunk, t
+            if log_progress:
+                progress.update(1)
```

Comment:
does the close() do this anyway?

Reply:
Discussed offline - this actually increments by 1, rather than setting to 100%